src/components/sections/Footer.js

```diff
@@ -1,80 +1,121 @@
+    link: 'https://graasp.eu',
   },
 ];
 
 const Footer = () => (
-  <StaticQuery
-    query={graphql`
-      query {
-        art_pot: file(
-          sourceInstanceName: { eq: "art" }
-          name: { eq: "customers_pot" }
-        ) {
-          childImageSharp {
-            fluid(maxWidth: 960) {
-              ...GatsbyImageSharpFluid_withWebp_tracedSVG
-            }
-          }
-        }
-      }
-    `}
-    render={data => (
-      <>
-        <Art>
-          <Img
-            fluid={data.art_pot.childImageSharp.fluid}
-            style={{ width: 480, maxWidth: '100%', marginBottom: -16 }}
-          />
-        </Art>
-        <FooterWrapper>
-          <StyledContainer>
-            <Copyright>
-              <h2>Absurd</h2>
-              <span>
-                Illustrations by
-                {` `}
-                <ExternalLink href="https://twitter.com/diana_valeanu">
-                  @diana_valeanu
-                </ExternalLink>
-              </span>
-            </Copyright>
-            <SocialIcons>
-              {SOCIAL.map(({ icon, link }) => (
-                <ExternalLink key={link} href={link}>
-                  <img src={icon} alt="link" />
-                </ExternalLink>
-              ))}
-            </SocialIcons>
-          </StyledContainer>
-        </FooterWrapper>
-      </>
-    )}
-  />
+  <>
+    <Section accent="secondary" style={{ paddingTop: 48, paddingBottom: 48 }}>
+      <StyledLogoContainer>
+        <div>
+          <h3 style={{ textAlign: 'center' }}>
+            The Graasp Association is supported by
+          </h3>
+          <LogoGrid>
+            {LOGOS.map(({ logo, link }) => (
+              <ExternalLink
+                key={link}
+                href={link}
+                style={{
+                  display: 'flex',
+                  justifyContent: 'center',
+                  alignItems: 'center',
+                }}
+              >
+                {logo()}
+              </ExternalLink>
+            ))}
+          </LogoGrid>
+        </div>
+      </StyledLogoContainer>
+    </Section>
+    <Art>
+      <GraaspLogoDark style={{ marginBottom: 12 }} />
+    </Art>
+    <FooterWrapper>
+      <StyledContainer>
+        <Copyright>© Graasp Association 2020</Copyright>
+        <SocialIcons>
+          {SOCIAL.map(({ icon, link }) => (
+            <ExternalLink key={link} href={link}>
+              <img src={icon} alt="link" />
+            </ExternalLink>
+          ))}
+        </SocialIcons>
+      </StyledContainer>
+    </FooterWrapper>
+  </>
 );
 
+const LogoGrid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 64px;
+  justify-items: center;
+  margin-top: 96px;
+
+  a {
+    svg {
+      width: 100%;
+    }
+  }
+
+  @media (max-width: ${props => props.theme.screen.sm}) {
+    grid-template-columns: 1fr;
+  }
+`;
+
+const StyledLogoContainer = styled(Container)`
+  margin: 0 auto;
+  position: relative;
```

Comment:
This removes the container from the normal "stack" of the page and causes it to float above the header.